README.md

```diff
@@ -205,29 +205,42 @@ Each one of them encapsulates the operations related to it (like listing, updati
 
 ### Webhooks
 
+#### `webhooks.create({ uid, tag, url, enable = false, secret, verifySSL })`
+
+- Create a webhook.
+- `uid`: Unique ID for the form.
+- `tag`: Unique name you want to use for the webhook.
+- `url`: Webhook URL.
+- `enabled`: `true` if you want to send responses to the webhook immediately. Otherwise, `false`.
+- `secret`: If specified, will be used to sign the webhook payload with HMAC SHA256, so that you can verify that it came from Typeform.
+- `verifySSL`: `true` if you want Typeform to verify SSL certificates when delivering payloads.
+
+#### `webhooks.delete({ uid, tag })`
+
+- Delete a webhook.
+- `uid`: Unique ID for the form.
```

Comment:
consistence between `of` and `for`

Reply:
It's effectively the verbiage from the [developers docs](https://developer.typeform.com/webhooks/reference/delete-webhook/).